docs/walkthrough/run-job.md

```diff
@@ -110,36 +110,3 @@ In general, you should choose a low learning rate (`1e-6` to `1e-4`) for fine-tu
 Otherwise, it could happen, that your model overfits on the training data and forgets the knowledge learned during pre-training.
 Similarly, two or three epochs (number of passes thorough the training data) are often enough for a fine-tuning job. 
 ```
-
-### Configuration of the miner
-
-To filter the instances in a batch that are used to calculate the loss, you can use miners.
-Finetuner allows you to use miners provided by the [PyTorch Metric Learning](https://kevinmusgrave.github.io/pytorch-metric-learning) framework.
-To select a specific miner, you can pass its name to the fit function, e.g., `AngularMiner`, `TripletMarginMiner`, ...
-
-Please note that the miner has to be compatible with the loss function you selected.
-For instance, if you choose to train a model with the `TripleMarginLoss`, you can use the `TripletMarginMiner`.
-While without this miner, all possible triples with an anchor, a positive, and a negative candidate are constructed, the miner reduces this set of triples.
-Usually, only triples with hard negatives are selected where the distance between the positive and the negative example is inside a margin of `0.2`.
-If you want to pass additional parameters to configure the miner, you can specify the `miner_options` parameter of the fit function.
-The example below shows how to apply hard-negative mining:
-
-```diff
-run = finetuner.fit(
-    ...,
-    loss='TripleMarginLoss',
-+   miner='TripletMarginMiner',
-+   miner_options={'margin': 0.3, 'type_of_triplets': 'hard'}
-)
-```
-
-The possible choices `type_of_triplets` are:
-
-+ `all`: Use all triplets, identical to no mining.
-+ `easy`: Use all easy triplets, all triplets that do not violate the margin.
-+ `semihard`: Use semi-hard triplets, the negative is further from the anchor than the positive.
-+ `hard`: Use hard triplets, the negative is closer to the anchor than the positive.
-
```

Comment:
How do I train with both 'hard' and 'semi-hard' items?

Reply:
currently it's not supported, so it's either/or, while it's easy to extend. Until someone request this feature